### DISCUSSION TITLE
專注力會傳染

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1538] [General]
你有沒有想過為什麼​
在咖啡廳工作唸書的效率​
好像總是比在自己家裡還要高？​

我自己在宿舍或家裡都完全沒辦法專心專注思考

科學家表示，這可能因為人的潛意識​
會受到旁人的呼吸、姿勢或氣味暗示​
是可以擴散到周圍，讓身邊的人也開始專心​

所以建議段考週需要讀書的同學們可以去k中或圖書館這種大家都很專注的地方

**Comment by PaulSu0905** [comment]
但我覺得更大的原因是社會壓力和責任感，在公共場所工作，我會感到一種社會壓力和責任感，不想被人看到我懶散或不專注。這種壓力可以促使我更加努力地專注於工作或學習。

**Comment by poweichen00** [comment]
對我來說在咖啡廳唸書算是一個相對安靜的環境跟舒適的氛圍，可以讓我更容易專心，不會讓我在家裡感覺要做不做的，很懶散。

**Comment by larry40305** [comment]
以前有聽說過把玩樂的地方跟讀書的地方分開有助學習，在讀書的地方不做其他事情有助其高專注度。

**Comment by Chiuhsianglin** [comment]
環境真的很影響一個人的專注度，只要有大考我都會選擇留在學校，因為待在家都會想睡覺、常常被其他的事情分心，但我也不太習慣待在太安靜的地方，所以我還在探索什麼地方是適合我讀書的。

**Comment by MaxChi870** [comment]
自己已經默認家裡是休閒放鬆的地方，且在家不受拘束，很常開書不到10分鐘，就跑去做其他雜事，在外讀書會因為旁人眼光，變得比較專心，以上為個人心得。

**Comment by aa103967622** [comment]
我很贊同這樣的看法，當周遭的人都在努力不懈地學習時，能夠刺激我的學習慾望，並且我會有所謂的自制力，去避免我去產生任何分心的念頭，在k中或是星巴克我都曾體會到這樣的氛圍，所有我很推大家在學習工作時，去這樣的地方。

**Comment by bestbanana295** [comment]
我覺得歡竟真的會影響一個人的專注力，在不同的環境底下人們會發展出不一樣的應對方式，所以說真的需要為自己找到一個可以專心的地方。讓自己科已近下心來好好讀書，不要一直划手機，冰且麄載專注力集中在想要讀的東西上。

**Comment by ashleyy-huang** [comment]
我覺得在我讀書的時候環境非常重要，若身邊有人很專心的在讀書也會使得我開始督促自己，像我就是一個不太喜歡在宿舍讀書的人一方面會吵，另一方面自己容易在房間怠惰，所以我也很喜歡去圖書館讀書

**Comment by Miinteaa** [comment]
很認同需要把休息的地方和讀書的地方分開的做法！只要給自己一個不是隨時都可以放鬆休息的環境，就能時刻提醒自己該認真了。但其實也蠻看心情的，有時候就算待在咖啡廳裡，旁邊的談話聲再吵都能專注，但有時候卻在安靜的圖書館都沒辦法靜下心讀書🥲

**Comment by zhiron544** [comment]
我覺得這篇文章很有趣，也很有道理。我自己也有過在不同地方讀書的經驗，發現在有人一起努力的地方，自己也會更有動力和專注力。我認為這是一種正向的社會影響，可以幫助我們提高學習效率和品質。不過，我也覺得每個人適合的環境可能不同，要找到自己最舒適和最有效的讀書方式。

**Comment by kai1660** [comment]
這一篇文章非'常有道哩，我就曾經親身體驗過，那是我第一次去K中，環境中非常安靜，大家都專心，不戴耳機都非常安靜，慢慢自己也逕入狀況，變得非常專心，所以我認同注意力是會傳染的。

**Comment by hollowha** [comment]
我覺得本來環境就是會影響個體的，所以有時候想自律改變自己，與其強迫自己，不如改變環境最直接輕鬆。

**Comment by RoyLin1014** [comment]
環境真的影響很大，如果要讀書去圖書館或是k中這種安靜且大家都在唸書的環境，一定會被影響而專心

**Comment by BOSENCHANG2004** [comment]
我覺得找一兩個同學組讀書會也非常的有用，同學彼此之間能夠互相督促讀書，自己也會因為有別人一起讀而不敢怠慢。

**Comment by SendaiHatsuki** [comment]
我也有一樣的感覺！自己在宿舍讀書常常會被床誘惑，或者一滑手機就停不下來
但一到咖啡廳或圖書館，看到周圍的人都很認真，自己也會進入讀書的狀態，真的蠻神奇的。
這種現象好像叫做「社會促進效應」，意思是我們在有他人在場時，行為表現會受到影響。像是看到別人認真，就會潛意識想要跟上，不想當那個在偷懶的人。
也有人說背景噪音像是咖啡廳的聲音，會剛好落在可以刺激注意力又不會太吵的程度，反而幫助集中。
所以我現在都會帶耳機播咖啡廳環境音在房間創造假裝的專注空間。

**Comment by edawdewv** [comment]
我也發現自己在咖啡廳或圖書館讀書的效率真的比在家好很多，可能就是因為周圍大家都在認真，潛意識裡會被感染，反而更容易專注。另外，家裡太舒適反而容易分心，像手機、電視、床都很誘惑人。去公共場所讀書，反而能把注意力集中在學習上。加上環境換了心情也會跟著不一樣，更有動力去完成目標。

**Comment by 12388833** [comment]
我也覺得專注力會傳染，我自己在家裡或宿舍常常一坐下來就忍不住滑手機、發呆，結果一整天過去什麼都沒讀進去。但只要換個地方，比如去K書中心、圖書館，甚至是人多的咖啡廳，看著大家都在認真用功，或許是不想跟別人不一樣，我也會莫名其妙地跟著進入狀態，效率比其在家好個好幾倍，我很推薦大家把家裡就當成休息的地方，把唸書跟休息的地方錯開。

**Comment by ryan8967** [comment]
個人認為家裡是最休閒放鬆的地方，不用通勤且在家不受拘束，讀起來比較舒適。雖然比較容易分心或跑去做其他雜事，但若能把注意力拉回就好。在外讀書會因為旁人眼光，變得比較專心但壓力也會大一點，通勤和準備也都需要時間。因此我平常會在家或宿舍讀，要大推進度才會到圖書館。

**Comment by LINSHIHMIN** [comment]
確實感覺在有其他人也在努力的環境下，自己也會被感染，雖然還是很會分心，但至少比自己一個人待著的時候還要更專注，可能是人都會有想比較的心態吧，有人在旁邊讀書的話，就會想著自己也要認真一點才行。但除了會被傳染專注力以外，還會有一種隱形的使命的感覺，是自己內在的動力，我本來就會覺得如果沒辦法把一件事做好那就乾脆不要開始，所以其實會默默地會對任何事都很認真，都想要做到好，有可能也是想符合社會期待，但我覺得有這種心態也沒有不好，就是不要太多極端就好。

**Comment by Pyra-0** [comment]
可能就是所謂的羊群效應吧，我自己也會有這個狀況，在做事的時候如果發現身邊的人也都在奮鬥，能在淺意識中下暗示說現在要專注，也能給深陷泥沼的自己一點安慰吧。但在那種環境下我還是挺不自在的，壓力也挺大，還是喜歡自己一個人做事。

**Comment by ghost** [comment]
不知道為什麼，身邊大部分人都在用功、而自己卻在休息的時候，真的會有一種罪惡感；而自己需要讀書，周遭的人都在休息的的時候，反而會無法認真下去。這種感覺就像是能量一樣，會從能量高往能量低的地方流動。

**Comment by wayne935** [comment]
我也有這樣的感覺，自己一個人的時候就會很放鬆，甚至躺在床上看書都有可能，最後一個不小心就會開始滑手機或不小心睡著。去圖書館就會無形中有種壓力讓自己覺得現在應該是要讀書的時間
